### PR TITLE
update to elm 0.19.1

### PR DESCRIPTION
**Pull request by objarni**
Thanks for a great little project! Got it running under elm 0.19.1, merge if you want to :)

### CODE REVIEW

**Comment by objarni** [comment]
Tested all elm files, they are running at least on Ubuntu 18.04 / Brave browser!

**Comment by w0rm** [comment]
Hey @objarni thanks for this! You've probably forgotten to commit the renamed `Circle.elm`!

**Comment by objarni** [comment]
@w0rm you are right, used the git commit -a -m 'comment' shorthand, it does not add untracked files files! Many thanks for a great repo, this will get me started with elm/webgl really quickly!

File: Circle.elm [hunk 1]
```diff
@@ -30,7 +30,7 @@ main =
 view : Float -> Html msg
 view t =
     WebGL.toHtml
-        [ width 400
+        [ width 800
```
Comment (w0rm) [inline Circle.elm]: Is this change really needed?

Comment (objarni) [reply Circle.elm]: No, just pushed removal of those changes.. I had started playing around, sorry :dancer:

**Comment by w0rm** [comment]
@objarni merged! You might also want to check https://github.com/nacmartin/elm-webgl-lessons by @nacmartin

**Comment by objarni** [comment]
Will do @w0rm thanks!
